exporter/prometheusexporter/prometheus_test.go

```diff
@@ -167,7 +167,7 @@ prometheus:
 	_ = res.Body.Close()
 	want := `# HELP test_this_one_there_where_ Extra ones
 # TYPE test_this_one_there_where_ counter
-test_this_one_there_where_{arch="x86",code="one",foo="bar",os="windows"} 99.0
+test_this_one_there_where_{arch="x86",code="one",foo="bar",os="windows"} 99
```

Comment:
Please revert this and keep the Prometheus version as before.

Reply:
I posted some comments earlier on why I am not enthusiastic about the Prometheus import change.

Reply:
Tried but that didn't work. Even if I reverted the Prometheus version in go.mod, after I ran `GO111MODULE=on go mod tidy` it will still be updated to `v0.9.3-0.20190127221311-3c4408c8b829`.